package-lock.json

```diff
@@ -0,0 +1,6644 @@
+{
```

Comment:
Could we drop the lock file out of git as it before?

Reply:
Done, but it's considered a bad practice :S